CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
   - The merge functionality is now embedded into `service.NewConfigProvider` (#4637).
 - Move `configtest.LoadConfig` and `configtest.LoadConfigAndValidate` to `servicetest` (#4606)
 - Builder: Remove deprecated `include-core` flag (#4616)
+- Move `compression.go` into `confighttp.go` (#4651)
```

Comment:
It doesn't seem like a breaking change. It's rather an additional public API / enhancement. Also please move it to "Unreleased" section and reword it to indicate what public API is exposed in which package.

Reply:
done. Thank you